### ISSUE TITLE
withColumnName() working only with uppercase column names

### ISSUE THREAD

**Issue by apoorva-sriv** [firegloves/MemPOI#54]
I have a `select Name from Table` query. When I use `.withColumnName("Name").withColumnDisplayName("First Name")`, `Name` is still the header in Excel. I need to use `.withColumnName("NAME")` to make it display `First Name`. Shouldn't it be case-insensitive and work in the first case too?

**Comment by firegloves** [comment]
@apoorva-sriv yes it should be case insensitive. Can I ask you to double-check which is the resulting column name from the result set by manually printing it? I suspect that the issue could derive from there

**Comment by firegloves** [comment]
I tested your case and I can't reproduce the issue.
This is what I did:

- Oracle XE 21.3
- MemPOI 1.8
- created a table with a field Name => this resulted automatically in a NAME field
- forced the NAME field to be Name (pascal case)
- executed this code

```
PreparedStatement prepStmt = connection.prepareStatement("SELECT * FROM MEMPOI.MYTABLE");

Workbook wb = new SXSSFWorkbook();

MempoiColumnConfig mempoiColumnConfig = MempoiColumnConfigBuilder.aMempoiColumnConfig()
                    .withColumnName("Name")
                    .withColumnDisplayName("First Name")
                    .build();

MempoiSheet mempoiSheet = MempoiSheetBuilder.aMempoiSheet()
                    .withPrepStmt(prepStmt)
                    .addMempoiColumnConfig(mempoiColumnConfig)
                    .build();
```

The resulting file contains the expected "First Name" string as the column header.
Please double-check if Oracle automatically capitalized the whole column name, I'm quite sure that this is the problem.

Otherwise please provide me with a way to reproduce your issue

**Comment by apoorva-sriv** [comment]
> Please double-check if Oracle automatically capitalized the whole column name, I'm quite sure that this is the problem.

Yes, [this](https://stackoverflow.com/questions/62822116/config-oracle-database-to-disable-uppercase-on-tables-and-columns-names/62822172#62822172) might likely be the problem, but that's why I raised the issue here: if both `select Name` and `select NAME` work in Oracle, then shouldn't both work with MemPOI too? Would it be a good idea to add a case with `toUpperCase` inside the definition of `withColumnName`?

**Comment by apoorva-sriv** [comment]
No, it's not blocking anything. I just thought it might help make the library more intuitive since, if it works with SQL, one might expect it would work here too.
